File: Programming-Fundamentals/Labs-And-Homeworks/Data-Types-and-Variables/Exercise/gladiator_expenses.py
number_of_lost_fights = int(input())
helmet_price = float(input())
sword_price = float(input())
shield_price = float(input())
armor_price = float(input())

total_helmet_broken = number_of_lost_fights // 2
total_sword_broken = number_of_lost_fights // 3
total_shield_broken = number_of_lost_fights // (2 * 3)
total_armor_broken = total_shield_broken // 2

expenses = total_helmet_broken * helmet_price \
           + total_sword_broken * sword_price \
           + total_shield_broken * shield_price \
           + total_armor_broken * armor_price

print(f"Gladiator expenses: {expenses:.2f} aureus")

File: JS-Front-End/JS-Front-End/Labs-And-Homeworks/JS-Syntax-Fundamentals/More-Exercises/gladiator-expenses.js
function gladiatorExpenses(lostFights, helmetPrice, swordPrice, shieldPrice, armorPrice) {
    let expenses = 0, shieldBreaks = 0;

    for (let i = 1; i <= lostFights; i++) {
        if (i % 2 === 0) expenses += helmetPrice;
        if (i % 3 === 0) expenses += swordPrice;
        if (i % 2 === 0 && i % 3 === 0) {
            expenses += shieldPrice;
            shieldBreaks++;
            if (shieldBreaks % 2 === 0) expenses += armorPrice;
        }
    }
    
    console.log(`Gladiator expenses: ${expenses.toFixed(2)} aureus`);
}

// Test Case

gladiatorExpenses(7, 2, 3, 4, 5);
gladiatorExpenses(23, 12.50, 21.50, 40, 200);